### PR TITLE
Import pdf generator

### PR DESCRIPTION
**Pull request by ngodfraind**
This could be added in the core section too because it no longer needs to be a plugin and there is not much code here.

### CODE REVIEW

File: import.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
 cmd("git checkout {$bundle}");
 cmd("git pull --no-tags {$bundle} master");
 cmd("git filter-branch -f --msg-filter 'sed \"1 s/^/[{$bundle}] /\"' HEAD");
-cmd("git checkout import-with-history");
+cmd("git checkout import-pdf-generator");
```
Comment (stefk) [inline import.php]: Probably not intended.

Comment (ngodfraind) [inline import.php]: Yes but it's ok. I usually overwrite this line to do the imports and have to commit the change otherwise the script will crash.

**Comment by stefk** [comment]
LGTM

**Comment by ngodfraind** [comment]
LGTM

**Comment by ngodfraind** [comment]
LGTM

Is it possible to remove the LGTM check if it has already been checked after a merge from the master ?

**Comment by stefk** [comment]
I don't think so, unfortunately.

**Pull request merged**